Airports/EGKK/SMAA.txt

```diff
@@ -157,4 +157,62 @@ EGKK Gatwick SMAA           S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00
+                            N051.07.51.280 W000.24.58.356 N051.07.42.496 W000.24.29.618
+                            N051.07.42.496 W000.24.29.618 N051.07.35.642 W000.23.59.576
+                            N051.07.35.642 W000.23.59.576 N051.07.30.792 W000.23.28.558
+                            N051.07.30.792 W000.23.28.558 N051.07.27.743 W000.22.51.593
+                            N051.07.27.743 W000.22.51.593 N051.07.27.394 W000.22.19.636
+                            N051.07.27.394 W000.22.19.636 N051.07.29.140 W000.21.47.794
+                            N051.07.29.140 W000.21.47.794 N051.07.32.963 W000.21.16.416
+                            N051.07.32.963 W000.21.16.416 N051.07.39.990 W000.20.40.852
+                            N051.07.39.990 W000.20.40.852 N051.07.48.141 W000.20.11.642
+                            N051.07.48.141 W000.20.11.642 N051.07.58.163 W000.19.43.950
+                            N051.07.58.163 W000.19.43.950 N051.08.09.942 W000.19.18.070
+                            N051.08.09.942 W000.19.18.070 N051.08.23.356 W000.18.54.296
+                            N051.08.23.356 W000.18.54.296 N051.08.38.253 W000.18.32.879
+                            N051.08.38.253 W000.18.32.879 N051.08.54.471 W000.18.14.058
+                            N051.08.54.471 W000.18.14.058 N051.09.11.834 W000.17.58.042
+                            N051.09.11.834 W000.17.58.042 N051.09.30.154 W000.17.44.999
+                            N051.09.30.154 W000.17.44.999 N051.09.49.223 W000.17.35.081
+                            N051.09.49.223 W000.17.35.081 N051.10.08.843 W000.17.28.396
+                            N051.10.08.843 W000.17.28.396 N051.10.32.132 W000.17.24.774
+                            N051.10.32.132 W000.17.24.774 N051.10.55.524 W000.17.25.710
+                            N051.10.55.524 W000.17.25.710 N051.11.18.672 W000.17.31.186
+                            N051.11.18.672 W000.17.31.186 N051.11.38.058 W000.17.39.437
+                            N051.11.38.058 W000.17.39.437 N051.11.56.793 W000.17.50.874
+                            N051.11.56.793 W000.17.50.874 N051.12.14.678 W000.18.05.382
+                            N051.12.14.678 W000.18.05.382 N051.12.31.511 W000.18.22.796
+                            N051.12.31.511 W000.18.22.796 N051.12.47.114 W000.18.42.927
+                            N051.12.47.114 W000.18.42.927 N051.13.01.312 W000.19.05.553
+                            N051.13.01.312 W000.19.05.553 N051.13.13.948 W000.19.30.429
+                            N051.13.13.948 W000.19.30.429 N051.13.24.885 W000.19.57.281
+                            N051.13.24.885 W000.19.57.281 N051.13.34.000 W000.20.25.815
+                            N051.13.34.000 W000.20.25.815 N051.13.41.196 W000.20.55.716
+                            N051.13.41.196 W000.20.55.716 N051.13.47.064 W000.21.31.893
+                            N051.13.47.064 W000.21.31.893 N051.13.50.121 W000.22.08.940
```

Comment:
Ahh yeah Luke's idea is a good one!
For the compiler to be happy about it, it will need the brackets and the word `region` in the ARC definition, like this:
```suggestion
;SMA Area 'C'
@ARC(region EGKK Gatwick SMAA centre N051.10.38.930 W000.22.30.290 radius 3.2)
```